metrics/report/report_dockerfile/lifecycle-time.R

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env Rscript
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+library(ggplot2)	# ability to plot nicely
+suppressMessages(suppressWarnings(library(tidyr))) # for gather()
```

Comment:
Could you either put these comments above the line, or do something about the alignment as it makes my head hurt :)